src/coord/src/coord.rs

```diff
@@ -651,7 +672,12 @@ impl<S: Append + 'static> Coordinator<S> {
                     )
                     .await?;
                 }
-                _ => (), // Handled in prior loop.
+                // Nothing to do for these cases
```

Comment:
@benesch Can you confirm this is the correct thing to do here?

Reply:
Yep, looks good!